src/Numerics/ODESolvers/AdditiveRungeKuttaMethod.jl

```diff
@@ -139,49 +144,64 @@ mutable struct AdditiveRungeKutta{
 
         # The code throughout assumes SDIRK implicit tableau so we assert that
         # here.
-        for is in 2:Nstages
-            @assert RKA_implicit[is, is] ≈ RKA_implicit[2, 2]
-        end
+        # for is in 2:Nstages
+        #     @assert RKA_implicit[is, is] ≈ RKA_implicit[2, 2]
+        # end
+
+        # NOTE: this is only for composing an ARK method with
+        # a MIS timestepper
+        # TODO: Clean this up
```

Comment:
should this `TODO` be handled?

Reply:
Sorry, yes, we already took care of it. Soon to be pushed together with other great suggestions from you :)